arangod/Graph/Graph.h

```diff
@@ -46,12 +46,23 @@ struct ServerDefaults;
 namespace graph {
 
 class EdgeDefinition {
+ private:
+  enum EdgeDefinitionType {
+    DEFAULT,
+    SMARTTOSMART,
+    SATTOSAT,
+    SMARTTOSAT,
+    SATTOSMART
+  };
+
  public:
   EdgeDefinition(std::string edgeCollection_, std::set<std::string>&& from_,
-                 std::set<std::string>&& to_)
+                 std::set<std::string>&& to_,
```

Comment:
I do not want to change it here, just wondering, why is it a `std::set` ?
Do we need this to be sorted?

Reply:
Think this one might be same as your mention below:
> True, but I would like to keep it sorted here to have sorted velocypack output (we're doing this for edgeColls, vertexColls as well)